tests/ci/integration/run_monit_integration.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - monit
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - MONIT_BUILD_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER="${SRC_ROOT}/MONIT_BUILD_ROOT"
+MONIT_SRC_FOLDER="${SCRATCH_FOLDER}/monit"
+MONIT_BUILD_FOLDER="${SCRATCH_FOLDER}/monit-aws-lc"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${SCRATCH_FOLDER}/aws-lc-install"
+
+function monit_build() {
+  ./bootstrap  
+  ./configure --with-ssl-static="${AWS_LC_INSTALL_FOLDER}"
+  make -j ${NUM_CPU_THREADS}
+}
+
+# Monit doesn't run any tests verifying ssl behavior, but it shouldn't hurt to run the brief tests.
+function monit_run_tests() {
+  pushd libmonit
+  # TimeTest will fail on a machine not in CET timezone.
+  # https://bitbucket.org/tildeslash/monit/src/def6b462259586358be3c86d76a299c80744df39/libmonit/test/TimeTest.c#lines-24
```

Comment:
is there anything important in TimeTest? if so, is it worth configuring this box to be in CET tz within this script?

Reply:
Not anything relevant to AWS-LC, they're time function tests to verify things are parsed correctly: https://bitbucket.org/tildeslash/monit/src/master/libmonit/test/TimeTest.c